shell/platform/android/io/flutter/plugin/editing/InputConnectionAdaptor.java

```diff
@@ -473,6 +485,63 @@ public boolean performEditorAction(int actionCode) {
     return true;
   }
 
+  @Override
+  public boolean commitContent(InputContentInfo inputContentInfo, int flags, Bundle opts) {
+      // Ensure permission is granted
+      if (BuildCompat.isAtLeastNMR1() && (flags & InputConnectionCompat.INPUT_CONTENT_GRANT_READ_URI_PERMISSION) != 0) {
```

Comment:
BuildCompat.isAtLeastNMR1() is deprecated. You can use the equivalent: https://developer.android.com/reference/androidx/core/os/BuildCompat#isAtLeastNMR1()